scripts/linters/pylint_extensions.py

```diff
@@ -1673,6 +1673,16 @@ class SingleLineCommentChecker(checkers.BaseChecker):
             'Please use a capital letter at the beginning of comment.',
             'no-capital-letter-at-beginning',
             'Please use capital letter to begin the content of comment.'
+        ),
+        'C0019': (
+            'Please use a single space at the beginning of pylint pragmas.',
+            'no-space-before-pylint-pragma',
```

Comment:
Perhaps use a more general name here and elsewhere since the pragmas we are considering are not specific to just pylint.

Reply:
@mzachlin This hasn't been changed? (I see down below you are using a new name `no-space-before-pragma` - please make sure you update msg-name at all applicable places to make sure it's working properly)